scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -371,7 +368,7 @@ case class Args(
       settingInScala2: String,
       settingInScala3Opt: Option[String]
   ): Option[String] = {
-    if (scalaVersion.isEmpty || scalaVersion.startsWith("2")) {
+    if (scalaVersion.value.isEmpty || scalaVersion.value.startsWith("2")) {
```

Comment:
add helper on `ScalaVersion`? Or `Ordering`?

Reply:
yes definitely. It was just to make everything compiles